stage1/bash/bash.mk

```diff
@@ -0,0 +1,16 @@
+# grab bash from the host
+
+include ../../makelib/lib.mk
+
+ISCRIPT := $(BUILDDIR)/install.d/10bash.install
+
+BASH_PATH := $(shell which $(BASH))
+
+.PHONY: install
+
+install:
+	@echo $(call dep-install-file,$(BASH_PATH)) > $(ISCRIPT)
+	@echo $(call dep-install-file,$(call find-file-so-deps,$(BASH_PATH))) >> $(ISCRIPT)
+# some search it in /usr/bin too
+	@echo $(call dep-install-dir,0755,/usr/bin) >> $(ISCRIPT)
```

Comment:
It would probably be better to make /bin, /sbin and /lib symlinks to /usr/bin/, /usr/sbin and /usr/lib respectively in stage1/aci/install.tpl.sh instead of having two copies of bash in aci.